src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
+import { Film } from '../../types/film';
+
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Film[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
```

Comment:
```suggestion
export const MoviesList: React.FC<Props> = (props) => (
const { movies } = props;
```
Let's stick to this kind of props usage